### PR TITLE
docs: Add signal information to getting started

### PR DESCRIPTION
**Pull request by hdost**
Closes prometheus/docs#167

Signed-off-by: Harold Dost <h.dost@criteo.com>

### CODE REVIEW

**Comment by hdost** [comment]
@juliusv  thoughts?

File: docs/getting_started.md [hunk 1]
```diff
@@ -250,3 +250,18 @@ scrape_configs:
 Restart Prometheus with the new configuration and verify that a new time series
 with the metric name `job_instance_mode:node_cpu_seconds:avg_rate5m`
 is now available by querying it through the expression browser or graphing it.
+
+## Reloading Configuration
+
+As mentioned in [configuration documentation](configuration/configuration.md) a
```
Comment (juliusv) [inline docs/getting_started.md:256]:
```suggestion
As mentioned in the [configuration documentation](configuration/configuration.md) a
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -250,3 +250,18 @@ scrape_configs:
...
+Prometheus instance can have its configuration reloaded without restarting the
+process by using the `SIGHUP` signal. If you're running on Linux this can be
+performed by using `kill -s SIGHUP <PID>` replacing `<PID>` with your Prometheus
```
Comment (juliusv) [inline docs/getting_started.md:259]:
```suggestion
performed by using `kill -s SIGHUP <PID>`, replacing `<PID>` with your Prometheus
```

File: docs/getting_started.md [hunk 3]
```diff
@@ -250,3 +250,18 @@ scrape_configs:
 Restart Prometheus with the new configuration and verify that a new time series
 with the metric name `job_instance_mode:node_cpu_seconds:avg_rate5m`
 is now available by querying it through the expression browser or graphing it.
+
+## Reloading Configuration
```
Comment (juliusv) [inline docs/getting_started.md:254]:
```suggestion
## Reloading configuration
```

File: docs/getting_started.md [hunk 4]
```diff
@@ -250,3 +250,18 @@ scrape_configs:
...
+process by using the `SIGHUP` signal. If you're running on Linux this can be
+performed by using `kill -s SIGHUP <PID>` replacing `<PID>` with your Prometheus
+process ID.
+
+## Shutting down your instance gracefully.
+
+While Prometheus does have recovery mechanisms in the case that there is an
+abrupt process failure it is recommend to use the `SIGTERM` signal to cleanly
+shutdown a Prometheus instance. If you're running on Linux this can be performed
+by using `kill -s SIGHUP <PID>` replacing `<PID>` with your Prometheus process ID.
```
Comment (juliusv) [inline docs/getting_started.md:267]:
```suggestion
by using `kill -s SIGHUP <PID>`, replacing `<PID>` with your Prometheus process ID.
```

**Comment by beorn7** [comment]
Randomly stumbled upon this. @hdost this seems almost done, how about finishing this by incorporating @juliusv 's comments?

**Comment by hdost** [comment]
@beorn7 🟢

File: docs/getting_started.md [hunk 5]
```diff
@@ -250,3 +250,18 @@ scrape_configs:
...
 with the metric name `job_instance_mode:node_cpu_seconds:avg_rate5m`
 is now available by querying it through the expression browser or graphing it.
+
+## Reloading configuration
+
+As mentioned in the [configuration documentation](configuration/configuration.md) a
+Prometheus instance can have its configuration reloaded without restarting the
+process by using the `SIGHUP` signal. If you're running on Linux this can be
+performed by using `kill -s SIGHUP <PID>`, replacing `<PID>` with your Prometheus
+process ID.
```
Comment (bwplotka) [inline docs/getting_started.md:260]: We can also use HTTP endpoint, but we can add that later. (:

File: docs/getting_started.md [hunk 6]
```diff
@@ -250,3 +250,18 @@ scrape_configs:
...
+
+## Shutting down your instance gracefully.
+
+While Prometheus does have recovery mechanisms in the case that there is an
+abrupt process failure it is recommend to use the `SIGTERM` signal to cleanly
+shutdown a Prometheus instance. If you're running on Linux this can be performed
+by using `kill -s SIGHUP <PID>`, replacing `<PID>` with your Prometheus process ID.
```
Comment (bwplotka) [inline docs/getting_started.md:267]: Missing new line at the end.

**Review by bwplotka — APPROVED** [review]
Thanks! LGTM

File: docs/getting_started.md (hunk 6 above)
Comment (hanjm) [inline docs/getting_started.md:267]: typo, it should be SIGTERM.

Comment (hdost) [reply docs/getting_started.md:267]: Good catch. I'll put out a patch.

**Pull request merged**